auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+            session_key=session_key or ''
+        )
+        
+        logger.info(
+            f"Verification record created for user: {user.username}, "
+            f"device: {device_name}"
+        )
+        
+        return verification
+    
+    @staticmethod
+    def refresh_access_token(refresh_token: str) -> Tuple[bool, Dict[str, any]]:
```

Ruff (0.12.2)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)